raml-maven-plugin/src/test/resources/lint-check/pom.xml

```diff
@@ -0,0 +1,29 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>uk.gov.justice.raml.test</groupId>
+    <artifactId>project-to-test</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>raml-maven-plugin</artifactId>
+                <groupId>uk.gov.justice.maven</groupId>
+                <version>@project.version@</version>
```

Comment:
Just  a question... Why is project.version wrapped in @ symbols and not ${project.version}?